webhook_features/src/features/summary_comment.rs

```diff
@@ -0,0 +1,631 @@
+}
+
+impl SummaryCommentFeature {
+    pub fn new() -> Self {
+        Self {
+            meta: FeatureMeta::new(
+                "Summary Comment",
+                "Creates a summary comment on pulls requests with an ACK tracker.",
+                vec![
+                    GitHubEvent::PullRequest,
+                    GitHubEvent::PullRequestReview,
+                    GitHubEvent::IssueComment,
+                ],
+            ),
+        }
+    }
+}
+
+#[async_trait]
+impl Feature for SummaryCommentFeature {
+    fn meta(&self) -> &FeatureMeta {
+        &self.meta
+    }
+
+    async fn handle(
+        &self,
+        ctx: &Context,
+        event: GitHubEvent,
+        payload: &serde_json::Value,
+    ) -> Result<()> {
+        let action = payload["action"]
+            .as_str()
+            .ok_or(DrahtBotError::UnknownEvent)?;
+
+        let repo_user = payload["repository"]["owner"]["login"]
+            .as_str()
+            .ok_or(DrahtBotError::KeyNotFound)?;
+
+        let repo_name = payload["repository"]["name"]
+            .as_str()
+            .ok_or(DrahtBotError::KeyNotFound)?;
+
+        let repo = Repository {
+            owner: repo_user.to_string(),
+            name: repo_name.to_string(),
+        };
+
+        println!("Handling event: {:?}", event);
+        println!("Action: {:?}", payload["action"]);
+        match event {
+            GitHubEvent::PullRequest if action == "opened" => {
+                create_summary_comment(payload, &ctx.octocrab).await?
+            }
+
+            GitHubEvent::PullRequest if action == "synchronize" => {
+                let pr_number = payload["number"]
+                    .as_u64()
+                    .ok_or(DrahtBotError::KeyNotFound)?;
+
+                refresh_summary_comment(ctx, repo, pr_number).await?
+            }
+            GitHubEvent::IssueComment if payload["issue"].get("pull_request").is_some() => {
+                let comment_author = payload["comment"]["user"]["login"]
+                    .as_str()
+                    .ok_or(DrahtBotError::KeyNotFound)?;
+
+                if comment_author == ctx.bot_username {
+                    return Ok(());
+                }
+
+                let pr_number = payload["issue"]["number"]
+                    .as_u64()
+                    .ok_or(DrahtBotError::KeyNotFound)?;
+                refresh_summary_comment(ctx, repo, pr_number).await?
+            }
+            GitHubEvent::PullRequestReview => {
+                let pr_number = payload["pull_request"]["number"]
+                    .as_u64()
+                    .ok_or(DrahtBotError::KeyNotFound)?;
+                refresh_summary_comment(ctx, repo, pr_number).await?
+            }
+            _ => {}
+        }
+        Ok(())
+    }
+}
+
+fn summary_comment_template(initial: bool, acks: Option<Vec<Review>>) -> String {
+    let mut comment = r#"
+### ACKs
+Please ACK this PR if you have reviewed it and believe it to be ready for merging.
```

Comment:
what about linking to the review guideline, as suggested by another reviewer previously?